docs/snippets/svelte/button-story-with-emojis.svelte.mdx

```diff
@@ -0,0 +1,15 @@
+```html
+<!-- Button.stories.svelte -->
+
+<Story name="Primary">
+    <Button background="#ff0" label="Button"/>
```

Comment:
indentation (and below)

Reply:
eslint is not executed on the svelte component files. I will fix that.